components/admin_console/billing/plan_details.tsx

```diff
@@ -196,6 +219,18 @@ const PlanDetails: React.FC = () => {
                 />
             </div>
         );
+        planDetailsDescription = goneBackUnderThreshold ? (
+            <div className='PlanDetails__description'>
+                <div className='PlanDetails__planDetailsName'>
+                    <FormattedMessage
+                        id='admin.billing.subscription.planDetails.planDetailsName.freeUpTo'
+                        defaultMessage='Free for up to {aboveUserLimit} users.'
```

Comment:
I believe the design indicates here should say "Free for up to 10 users" so we should use here `userLimit`
```suggestion
                        defaultMessage='Free for up to {userLimit} users.'
```